package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "Just a leaderboard web app",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/karelvanoordt/Leaderboard.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/karelvanoordt/Leaderboard/issues"
+  },
+  "homepage": "https://github.com/karelvanoordt/Leaderboard#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.5.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.25.3",
+    "hint": "^6.1.8",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.62.2",
+    "webpack-cli": "^4.9.1"
```

Comment:
- [ ] webpack-dev-server is missing in this devDependencies section, please add it because you will most likely need it in the next milestones.